examples/resource-manager.v2.test.js

```diff
@@ -0,0 +1,248 @@
+/**
+ * @jest-environment node
+ */
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const ResourceManagerV2 = require('../dist/resource-manager/v2');
+const { readExternalSources } = require('ibm-cloud-sdk-core');
+const authHelper = require('../test/resources/auth-helper.js');
+
+//
+// This file provides an example of how to use the Resource Manager service.
+//
+// The following configuration properties are assumed to be defined:
+// RESOURCE_MANAGER_URL=<service base url>
+// RESOURCE_MANAGER_AUTH_TYPE=iam
+// RESOURCE_MANAGER_APIKEY=<IAM apikey>
+// RESOURCE_MANAGER_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+// RESOURCE_MANAGER_TEST_QUOTA_ID=<Quota Id used in testing>
+// RESOURCE_MANAGER_TEST_USER_ACCOUNT_ID=<User Id used in testing>
+//
+// These configuration properties can be exported as environment variables, or stored
+// in a configuration file and then:
+// export IBM_CREDENTIALS_FILE=<name of configuration file>
+//
+const configFile = 'resource_manager_v2.env';
+
+const describe = authHelper.prepareTests(configFile);
+
+// Save original console.log and console.warn
+const originalLog = console.log;
+const originalWarn = console.warn;
+
+// Mocks for console.log and console.warn
+const consoleLogMock = jest.spyOn(console, 'log');
+const consoleWarnMock = jest.spyOn(console, 'warn');
+
+describe('ResourceManagerV2', () => {
+  
+  // begin-common
+  
+  const resourceManagerService = ResourceManagerV2.newInstance({});
```

Comment:
We should create another service instance for the delete operation. Let's call it `deleteResourceManagerService` and the use the service name `ALT_RESOURCE_MANAGER`.